ui/components/src/Legend/Legend.stories.tsx

```diff
@@ -115,97 +118,55 @@ export default meta;
 
 type Story = StoryObj<typeof Legend>;
 
-export const Bottom: Story = {
-  args: {
-    options: {
-      position: 'Bottom',
-    },
-  },
-};
-
-export const Right: Story = {
-  args: {
-    width: 200,
-    height: 300,
-    options: {
-      position: 'Right',
-    },
-  },
-};
-
 /**
```

Comment:
Sorry the diffs are a little rough in the storybook. I reordered a few of the stories and did some other cleanup to make the docs look nicer for consumers, but it made a yucky code diff.